### PR TITLE
KDL-841 widget updates

### PR DESCRIPTION
**Pull request by tsyvinska**

### CODE REVIEW

File: src/containers/SmbFlow/SmbFlow.jsx [hunk 1]
```diff
@@ -143,6 +151,19 @@ class SmbFlow extends BaseMobileFlow {
         return Promise.resolve();
       }
 
+      if (err.response.status === 403 && err.response.data.detail === 'User does not have available calculations.') {
+        const {
```
Comment (ratnev) [inline src/containers/SmbFlow/SmbFlow.jsx:155]: Just a suggestion:

make it as const (NO_AVAILABLE_CALCULATIONS_TEXT = 'User does not have available calculations.') e.g.

File: src/containers/WeightContainer/WeightContainer.jsx [hunk 2]
```diff
@@ -152,10 +238,13 @@ class WeightContainer extends Component {
...
     } else {
       setWeight(+value);
+      setWeightLb(getWeightLb(+value));
     }
 
     this.setState({
       weightValue: value,
+      minWeight: units === 'cm' ? 30 : 66,
+      maxWeight: units === 'cm' ? 200 : 441,
     });
```
Comment (ratnev) [inline src/containers/WeightContainer/WeightContainer.jsx:248]: make it variables as:

const MIN_WEIGHT_METRIC = 30;
const MIN_WEIGHT_IMPERIAL = 66;
const MAX_WEIGHT_METRIC....

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -103,30 +105,39 @@ class Header extends Component {
       } else {
         window.location = returnUrl;
       }
+    } else {
+      window.location = returnUrl || 'https://3dlook.ai/mobile-tailor/';
     }
```
Comment (ratnev) [inline src/components/Header/Header.jsx:110]: make it const e.g.

const DEFAULT_RETURN_URL = 'https://3dlook.ai/mobile-tailor/';

File: src/containers/SmbFlow/SmbFlow.jsx (hunk 1 above)
Comment (tsyvinska) [reply src/containers/SmbFlow/SmbFlow.jsx:155]: yep, the suggestion is good, but I'd prefer to leave as it is, just to keep curent files states as in a private-mtm-widget repo. What do you think?

**Review by ratnev — APPROVED** [review]

**Pull request merged**